tests/_utils/config.py

```diff
@@ -101,17 +101,31 @@ def test_files(self) -> tuple[Path, ...]:
 
         if self.examples:
             examples = (
-                path
-                for path in Path("examples").glob("*.py")
-                if str(path) not in SKIPPED_EXAMPLES
+                path.relative_to(self.root_dir)
+                for path in Path(self.root_dir.joinpath("examples")).glob(
+                    "*.py"
+                )
+                if str(path.relative_to(self.root_dir)) not in SKIPPED_EXAMPLES
             )
             files.extend(sorted(examples))
 
         if self.integration:
-            files.extend(sorted(Path("tests/integration").glob("*.py")))
+            integration_tests = (
+                path.relative_to(self.root_dir)
+                for path in Path(
+                    self.root_dir.joinpath("tests/integration")
+                ).glob("*.py")
```

Comment:
explicit `Path` also unnecessary here since `self.root_dir` is a `Path`

Reply:
Without Path, it would raise AttributeError: 'PurePosixPath' object has no attribute 'glob'.
As self.root_dir.joinpath("examples") is 'PurePosixPath', Path is necessary here.